<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DIInterruptClass</class>
 <widget class="QDialog" name="DIInterruptClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>277</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>436</width>
    <height>277</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>436</width>
    <height>277</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>DI Interrupt</string>
  </property>
  <widget class="QPushButton" name="btnConfig">
   <property name="geometry">
    <rect>
     <x>346</x>
     <y>230</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </widget>
  <widget class="QFrame" name="background">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>-2</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame#background{background-image:url(:/DIInterrupt/Resources/Background.png)}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="eventDataList">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>38</y>
     <width>321</width>
     <height>221</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>321</width>
     <height>221</height>
    </size>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="showGrid">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>0</number>
   </property>
   <property name="columnCount">
    <number>0</number>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QLabel" name="gifViewer">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>5</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStop">
   <property name="geometry">
    <rect>
     <x>346</x>
     <y>165</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStart">
   <property name="geometry">
    <rect>
     <x>346</x>
     <y>117</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>23</height>
    </size>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
